src/app/containers/Video/index.stories.jsx

```diff
@@ -0,0 +1,17 @@
+import { storiesOf } from '@storybook/react'; // eslint-disable-line
+import {
+  VideoWithCaption,
+  VideoWithoutCaption,
+  VideoAmpWithCaption,
+  VideoAmpWithoutCaption,
+} from './fixtureData';
+import AmpDecorator from '../../helpers/storybook/ampDecorator';
+
+storiesOf('Video', module)
```

Comment:
Namespacing this to VideoContainer will separate out the container stories from the Component Story. 
```suggestion
storiesOf('Video Container', module)
```